bedrock/mozorg/templates/mozorg/home/home-en-election.html

```diff
@@ -10,11 +10,16 @@
 <div class="mozilla-content">
   <div class="mzp-l-content">
     <div class="mzp-l-card-hero">
+      {% if switch('firefox-election-build') %}
+          {% set bundle-title = 'Get the Firefox Election Bundle' %}
```

Comment:
Exception Value: `("expected token 'end of statement block', got '-'",)`

Looks like this has to be an underscore. `bundle_title` fixes it for me locally.